vars/acePushConfigToGit.groovy

```diff
@@ -32,6 +32,7 @@ void call(Map opts = [:]) {
     git config --global user.email jenkins@tietoevry.com
     git config --global user.name "Jenkins the autonomous"
 
+    rm -rf gitops
```

Comment:
Please create a unique folder name using mktemp.

Reply:
Change to use mktemp folder for gitops repo clone